client/src/components/BlipDetailSheetComponent.jsx

```diff
@@ -57,20 +57,19 @@ class BlipDetailSheetComponent extends React.Component {
         } else {
             this.setState({valid: true});
             const modifiedComments = this.state.comments;
-            const timestamp = new Date().toLocaleString();
             writeCommentService.addComment({
                 autor: this.state.newCommentAutor,
                 text: this.state.newCommentText,
                 meinung: this.state.newMeinung,
-                zeit: timestamp,
+                zeit: new Date(),
```

Comment:
im dateformat sind doch aber zu viele infos mit dabei, oder? also kein user möchte timezone information sehen sondern einfach die zeit in seiner zeitzone, deswegen hatten wir bewusst toLocaleString() verwendet wenn ich mich recht erinnere... aber dazu kann @jennybohn mehr sagen denke ich, sie hatte das gemacht ursprünglich. außerdem clashed das komplett die sortierung der zeiten denke ich, es ist wichtig, dass immer exakt das format `DD/MM/YYYY, HH:MM:SS` in die datenbank kommt

Reply:
ich habe mal einen anderen ansatz gefunden, kann ich das einfach noch mal pushen oder soll ich dafür nen neuen branch aufmachen?

Reply:
nee kannst einfach auf den Branch pushen